Algorithm.Python/BasicTemplateOptionTradesAlgorithm.py

```diff
@@ -21,51 +21,47 @@
 from QuantConnect.Algorithm import *
 from datetime import timedelta
 
+### <summary>
+### This example demonstrates how to add options for a given underlying equity security.
+### It also shows how you can prefilter contracts easily based on strikes and expirations.
+### It also shows how you can inspect the option chain to pick a specific option contract to trade.
+### </summary>
+### <meta name="tag" content="using data" />
+### <meta name="tag" content="options" />
+### <meta name="tag" content="filter selection" />
 class BasicTemplateOptionTradesAlgorithm(QCAlgorithm):
-    ''' This example demonstrates how to add options for a given underlying equity security.
-        It also shows how you can prefilter contracts easily based on strikes and expirations.
-        It also shows how you can inspect the option chain to pick a specific option contract to trade.'''
 
     def Initialize(self):
         self.SetStartDate(2015, 1, 1)
         self.SetEndDate(2015, 3, 20)
         self.SetCash(100000)
 
-        equity = self.AddEquity("GOOG", Resolution.Minute)
-        option = self.AddOption("GOOG", Resolution.Minute)
-        self.symbol = option.Symbol
+        option = self.AddOption("GOOG")
 
         # set our strike/expiry filter for this option chain
         option.SetFilter(-2, +2, timedelta(0), timedelta(30))
-        
-        equity.SetDataNormalizationMode(DataNormalizationMode.Raw)
-        
+
         # use the underlying equity as the benchmark
-        self.SetBenchmark(equity.Symbol)
+        self.SetBenchmark("GOOG")
 
 
     def OnData(self,slice):
         if not self.Portfolio.Invested:
             for kvp in slice.OptionChains:
                 chain = kvp.Value
                 # find the second call strike under market price expiring today
-                contracts = sorted(sorted(chain, key = lambda x: abs(chain.Underlying.Price - x.Strike)), 
```

Comment:
In order to find the most at-the-money contract, should we sort the chain by expiry first then by the absolute difference between underlying price and the strike?

Reply:
@mchandschuh I asked @jingwu74 to check whether the change would make a difference, she test it and didn't find it necessary.